apps/src/netsim/NetSimRouterNode.js

```diff
@@ -633,6 +654,14 @@ NetSimRouterNode.prototype.routeMessage_ = function (message, myWires) {
     return;
   }
 
+  // Automatic DNS: requests to address zero hit the "automatic DNS" system
+  // and generate responses.
+  // TODO (bbuchanan): Send to a real auto-dns node
```

Comment:
TODO (apologies if this is annoying, i just like to point out TODOs in case folks forgot that they still need to address them)